### ISSUE TITLE
enforce-consistent-line-wrapping: classesPerLine counts total classes instead of classes per line

### ISSUE THREAD

**Issue by fkoepke** [sergioazoc/oxlint-tailwindcss#111]
The `classesPerLine` option has two problems that make it incompatible with the multiline convention:

```tsx
<div className={`
  class1 class2 class3
  class4 class5 class6
`} />
```

### Problem 1: check counts total classes, not classes per line

The rule checks `classes.length > classesPerLine`, where `classes` is the full flat list across all lines. This produces false positives for already-correctly-wrapped strings.

The error message says **"Too many classes on a *single line*"** — confirming the intent is per-line, not total.

```ts
// 6 classes, 3 per line — no line exceeds classesPerLine: 4 — should be VALID, currently reports ✗
{
  code: 'const className = `\n  class1 class2 class3\n  class4 class5 class6\n`',
  filename: 'test.tsx',
  options: [{ classesPerLine: 4 }],
},
```

### Problem 2: autofix does not produce or preserve the `\n  classes\n` format

The autofix builds `chunks.join('\n' + indent)` where `indent` is the backtick's column position. It produces inline continuation without leading or trailing newlines:

```ts
// Input (5 classes, classesPerLine: 2):
const className = `class1 class2 class3 class4 class5`
//                 ^ backtick at column 18

// Current autofix output (18-space indent, no leading/trailing newlines):
const className = `class1 class2
                  class3 class4
                  class5`

// Expected autofix output:
const className = `
  class1 class2
  class3 class4
  class5
`
```

When the autofix runs on an already-multiline string, the leading and trailing newlines are also lost:

```ts
// Input (already multiline, too many per line):
const className = `
  class1 class2 class3 class4 class5
`

// Current autofix output (leading/trailing newlines stripped):
const className = `class1 class2
                  class3 class4
                  class5`

// Expected autofix output (format preserved):
const className = `
  class1 class2
  class3 class4
  class5
`
```

**Comment by sergioazoc** [comment]
Thanks @fkoepke, both problems were real and you framed them well.

`classesPerLine` was counting the flat total across all lines instead of per line, so an
already-wrapped block was a false positive. And the autofix was flattening the string, losing the
leading and trailing newlines.

Both fixed in #113: it counts per line now, and the autofix wraps into the block convention
non-destructively (only over-budget lines re-wrap, conforming lines stay verbatim, and the base
indent is read from the source line so it nests correctly inside JSX). The output matches the shape
you asked for. Shipping in v1.8.0.
